### ISSUE TITLE
New scripting style

### ISSUE THREAD

**Issue by yhara** [biwascheme/biwascheme#159]
## Old style

```html
<script src="biwascheme.js">
(display "Hello")
</script>
```

- pros: Short
- cons: Filename matters 

## New style

```html
<script src="biwascheme.js"></script>
<script type="text/biwascheme">
(display "Hello")
</script>
```

- pros:
  - You don't need to care about .js file name
  - You can split Scheme code into several script tags

**Comment by yhara** [comment]
Actually the new style is already available since #82. The example on the website should be updated before releasing 0.7.0.

**Comment by yhara** [comment]
> The example on the website should be updated before releasing 0.7.0.

Done.
